android/app/src/main/java/com/ecomap/ecomap/MainActivity.kt

```diff
@@ -487,6 +555,51 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
         containerInfoWindowView.visibility = View.GONE
     }
 
+    /**
+     * Gets the current list of containers that the user has bookmarked.
+     */
+    private fun getUserContainerBookmarks() {
+        userContainerBookmarks.clear()
+
+        // Helper function to handle a successful response.
+        val handleSuccess = fun(paginatedContainers: ContainersPaginated) {
+            if (isFinishing || isDestroyed) {
+                return
+            }
+
+            for (container in paginatedContainers.containers) {
+                userContainerBookmarks[container.id] = container
+            }
+        }
+
+        // Execute the request to get all existing user container bookmarks and add them to the list.
+        val request = ApiClient.listUserContainerBookmarks(
+            userID,
+            REQUEST_LIST_CONTAINER_LIMIT,
+            0,
+            token,
+            { paginatedContainers ->
+                val remainingRequest = paginatedContainers.total / REQUEST_LIST_CONTAINER_LIMIT
+                for (i in 1..remainingRequest) {
+                    ApiRequestQueue.getInstance(applicationContext).add(
+                        ApiClient.listUserContainerBookmarks(
+                            userID,
+                            REQUEST_LIST_CONTAINER_LIMIT,
+                            REQUEST_LIST_CONTAINER_LIMIT * i,
+                            token,
+                            { handleSuccess(it) },
+                            { Common.handleVolleyError(this, this, it) }
+                        )
+                    )
+                }
+
+                handleSuccess(paginatedContainers)
+            },
+            { error -> Common.handleVolleyError(this, this, error) })
```

Comment:
The line 591 uses `it` instead of `error`. Please update one or the other to keep both the same.

Reply:
Sure, this was a leftover from when there was no `Common.handleVolleyError`